Assets/_Project/Scripts/UserInterface/Components/AppearanceChangingButton.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Serialization;
+using UnityEngine.UI;
+
+namespace PolSl.UrbanHealthPath.UserInterface.Components
+{
+    [RequireComponent(typeof(Button))]
+    public class AppearanceChangingButton : MonoBehaviour
+    {
+        [Serializable] public struct ButtonAppearance
```

Comment:
Any reason why this is a struct?

Reply:
It has to be a struct to keep to elements connected in a list.